stable/kommander/templates/federated-addons.yaml

```diff
@@ -0,0 +1,161 @@
+{{- if .Values.federate.addons }}
+---
+apiVersion: core.kubefed.io/v1beta1
+kind: FederatedTypeConfig
+metadata:
+  finalizers:
+    - core.kubefed.io/federated-type-config
+  generation: 1
+  name: addons.kubeaddons.mesosphere.io
+  namespace: {{ .Release.Namespace }}
+spec:
+  federatedType:
+    group: types.kubefed.io
+    kind: FederatedAddon
+    pluralName: federatedaddons
+    scope: Namespaced
+    version: v1beta1
+  propagation: Enabled
+  targetType:
+    group: kubeaddons.mesosphere.io
+    kind: Addon
+    pluralName: addons
+    scope: Namespaced
+    version: v1alpha1
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  generation: 1
+  name: federatedaddons.types.kubefed.io
```

Comment:
Should this be in our own domain space?

Reply:
Typically, most CRDs are named for the organization that's responsible for maintaining it, which would imply that `federatedaddons.types.kubefed.io` would be maintained by kubernets/sig-multicluster. Since, afaict, that's our own CRD, it seems to me it should be suffixed with `mesosphere.io` (or maybe even `d2iq.io`. Not sure if the rest are ever going to be changed since that's a backward compatibility problem.). But then again, maybe there's a reason they have to be suffixed with `kubefed.io` that I'm not aware of.

Reply:
Oh I understand now, thanks for clarifying! This isn't something we own, we are enabling a federated resource with these files, we aren't creating a crd, it is confusing admittedly.

Reply:
not sure if it'll break kubefed if we change the domain space. these are generated via kubefed, which needs to follow kubefed specs. so i think it's ok to kubefed to own it.